config/routes.rb

```diff
@@ -108,8 +108,11 @@
       get "/replace-or-update-induction-tutor", to: "schools/replace_or_update_induction_tutor#show"
       post "/replace-or-update-induction-tutor", to: "schools/replace_or_update_induction_tutor#choose"
       resources :cohorts, controller: "schools/cohorts", only: :index
+      resources :participants, controller: "schools/participants", only: :index
     end
 
+    resources :participants, only: :show
```

Comment:
These should probably be a single resource

Reply:
Why? There is more than one participant - am I misunderstanding resource vs resources?